doc/2/api/protocols/websocket/index.md

```diff
@@ -63,3 +63,11 @@ The listening port can be modified under the `server.port` section of the [confi
 HTTP and WebSocket protocols share the same underlying server instance.  
 Modifying the listening port will impact these two protocols.
 :::
+
+<SinceBadge version="2.10.0"/>
+:::info
+We needed the client to be able to ping Kuzzle to have a keep-alive system.
+Through the browser, that was not possible to send real formatted pings websocket frames since there is no API implemented for that unlike the one of Node.
+We needed something higher level.
+That is the reason why when you are using the SDK Javascript, the ping/pong between Kuzzle and your client is a custom message interpreter.
+:::
```

Comment:
You need to fully document the feature, including the expected application-level PING format, and the corresponding PONG response.

Also, I would change how this documentation is worded (to be put in a chapter of its own, and not as a side-note):

```suggestion
:::info
Though Kuzzle's WebSocket server is fully compliant with the RFC6455, meaning (among other things) that Kuzzle will respond to PING packets with standard PONG ones, an additional PING request has been added in the protocol's application layer.

This application-level PING has been especially added for web browsers, which don't allow sending PING packets. This can be troublesome if a web application needs to know if a connection has been severed, or if Kuzzle is configured to be in passive mode (i.e. it won't send PING requests by itself, and will close sockets if they are idle for too long).

When run in a browser, our Javascript SDK uses that feature for its keep-alive mechanism.
:::
```